packages/nextjs/app/builders/0x04cdfb463600a29e8e93053b88daa11f54379907/page.tsx

```diff
@@ -0,0 +1,467 @@
+"use client";
+
+import React, { useEffect, useState } from "react";
+import Image from "next/image";
+import {
+  Award,
+  Check,
+  CheckCircle2,
+  Code2,
+  Copy,
+  Gem,
+  Globe,
+  Layers,
+  Lock,
+  Palette,
+  ScrollText,
+  Shield,
+  TrendingDown,
+  TrendingUp,
+} from "lucide-react";
+import { NextPage } from "next";
+import { useTheme } from "next-themes";
+
+const WALLET_ADDRESS = "0x04cdfb463600a29e8e93053b88daa11f54379907";
+const WALLET_SHORT = "0x04cD...9907";
+
+const customStyles = `
+  @keyframes scroll {
+    0% { transform: translateX(0); }
+    100% { transform: translateX(-50%); }
+  }
+  .animate-scroll {
+    animation: scroll 20s linear infinite;
+  }
+  .animate-scroll:hover {
+    animation-play-state: paused;
+  }
+
+  .uiverse-card {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    gap: 15px;
+    padding: 10px;
+  }
+  .socialContainer {
+    width: 48px;
+    height: 48px;
+    border-radius: 12px;
+    background-color: rgb(44, 44, 44);
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    overflow: hidden;
+    transition-duration: .3s;
+    border: 1px solid rgba(255,255,255,0.05);
+  }
+  .socialSvg {
+    width: 20px;
+    height: 20px;
+    transition: all 0.3s;
+    fill: white; 
+    color: white;
+  }
+  .socialContainer:hover .socialSvg {
+    animation: slide-in-top 0.3s both;
+  }
+  .containerGithub:hover { background-color: #333; }
+  .containerX:hover { background-color: #000; }
+  .containerDiscord:hover { background-color: #5865F2; }
+  .containerTelegram:hover { background-color: #0088cc; }
+  .containerFarcaster:hover { background-color: #8A63D2; }
+  .containerSpeedRun:hover { background-color: #8B5CF6; }
+  .socialContainer:active {
+    transform: scale(0.9);
+    transition-duration: .3s;
+  }
+  @keyframes slide-in-top {
+    0% { transform: translateY(-50px); opacity: 0; }
+    100% { transform: translateY(0); opacity: 1; }
+  }
+`;
+
+function SkillTag({ label, icon, isDark }: { label: string; icon: React.ReactNode; isDark: boolean }) {
+  const containerStyle = isDark
+    ? "bg-slate-800/40 border-slate-700/50 hover:border-blue-500/50 hover:bg-slate-800/60 hover:shadow-[0_0_15px_rgba(59,130,246,0.2)] text-slate-300"
+    : "bg-[#e2e8f0] border-slate-300 hover:border-blue-400 hover:shadow-md text-slate-700 hover:text-blue-700";
```

Comment:
As I said: You should only have hover effects for interactive elements. This hover effect misleads the user into thinking they can click it to navigate.